### PR TITLE
Added postman collection for payouts and 3pp

### PR DESCRIPTION
**Pull request by sanj-kv**
## Context

The payouts api collection was outdated. So, added a folder that focuses on payouts and 3pp apis specifically.

JIRA ticket - https://transferwise.atlassian.net/jira/software/c/projects/SE/boards/1090?modal=detail&selectedIssue=SE-895
## Checklist
- [X] Change meets or does not compromise the [Baseline Security Requirements](https://transferwise.atlassian.net/wiki/spaces/EKB/pages/434929973/Baseline+Security+Requirements)

### CODE REVIEW

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,1728 @@
...
+									"raw": "{{host}}/oauth/token",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"oauth",
+										"token"
+									]
+								},
+								"description": "This endpoint is used to link an existing TransferWise account to a bank account. It is used as part of the redirection flow that allows the customer to sign into their TransferWise account and authorise the linking of their profile."
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Description needs to be updated to "Wise" and maybe more payouts-appropriate?

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,1728 @@
...
+									"raw": "{{host}}/oauth/token",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"oauth",
+										"token"
+									]
+								},
+								"description": "This endpoint is used to generate a new access token if the previous one has expired. The refresh token is initially generated when creating a new user or when linking an existing TransferWise account."
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Description needs update

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,1728 @@
...
+										"value": "Bearer {{token}}"
+									},
+									{
+										"key": "Content-Type",
+										"value": "application/json"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n\t\"sourceCurrency\": \"GBP\",\n\t\"targetCurrency\": \"USD\",\n\t\"sourceAmount\": 100,\n\t\"targetAmount\": null,\n\t\"profile\": <owning profile id>\n}"
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
									"raw": "{\n\t\"sourceCurrency\": \"GBP\",\n\t\"targetCurrency\": \"USD\",\n\t\"sourceAmount\": 100,\n\t\"targetAmount\": null,\n\t\"profile\": {{business-profile-id}}\n}"
```

This should now work, at the time there was a postman bug

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,1728 @@
...
+												"quotes",
+												"{{new-quote-id}}",
+												"account-requirements"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "2. Update form if any field require refresh",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
									"name": "2. Update form if any field requires refresh",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 5]
```diff
@@ -0,0 +1,1728 @@
...
+											"path": [
+												"v1",
+												"accounts"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "3. Create account - SGD recipient example",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
									"name": "EXAMPLE: Create account - SGD recipient",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 6]
```diff
@@ -0,0 +1,1728 @@
...
+											"path": [
+												"v1",
+												"accounts"
+											]
+										}
+									},
+									"response": []
+								},
+								{
+									"name": "3. Create account - Global USD example",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
									"name": "EXAMPLE: Create account - Global USD",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 7]
```diff
@@ -0,0 +1,1728 @@
...
...d the transfer's funds - see the [transfer status here](https://transferwise.github.io/api-docs-banks/#transferwise-for-banks-api-integration-guide-track-transfer-status) for cases when this may happen.\n\nA refund recipient is a person or institution where we will refund transfer funds to. This is not a mandatory step if your bank is using a fast local payment network as we can usually infer the refund recipient from the bank transaction that funded the transfer.\n\nBut if your bank is using a slow domestic payment network and a \"prefunded\" model, we may try to refund before we receive the funds in our bank transactions. In these instances, we require banks to inform us what are the details where to refund transfers. In order to do this we have current API\n\nThis API is identical to the regular recipient account creation. The refund recipient account id returned here is needed when creating transfers in step 3 as the field sourceAccount.\n\nThe format of the request payload for refund recipient creation will be different depending on the currency your bank will send transfers from. The above example is for GBP only. We can provide the correct format for your bank upon request."
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Do we have a link to this in payouts guide? Probably needs rewording also.

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 8]
```diff
@@ -0,0 +1,1728 @@
...
+												{
+													"key": "Connection",
+													"value": "keep-alive"
+												},
+												{
+													"key": "Server",
+													"value": "nginx/1.14.0 (Ubuntu)"
+												},
+												{
+													"key": "x-request-id",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Remove?

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 9]
```diff
@@ -0,0 +1,1728 @@
...
+											"key": "Connection",
+											"value": "keep-alive"
+										},
+										{
+											"key": "Server",
+											"value": "nginx/1.14.0 (Ubuntu)"
+										},
+										{
+											"key": "x-request-id",
+											"value": "535d9301-fd29-9470-a43b-27c8761465bb"
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Remove headers?

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 10]
```diff
@@ -0,0 +1,1728 @@
...
+									"path": [
+										"v1",
+										"transfer-requirements"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "4. Generate GUID for idempotency",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
							"name": "4. Generate GUID for idempotency - payouts only",
```

This isn't required for 3PP, not sure if we can word it better though...

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 11]
```diff
@@ -0,0 +1,1728 @@
...
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"method": "POST",
+								"header": [],
+								"body": {
+									"mode": "raw",
+									"raw": "{\n    \"targetAccount\": {{new-recipient-id}},\n    \"quote\": \"{{new-quote-id}}\",\n    \"originalTransactionId\": \"<idempotency guid>\",\n    \"details\": {\n        \"reference\": \"Holiday\"\n    },\n    \"originator\": {\n        \"legalEntityType\": \"PRIVATE\",\n        \"reference\" : \"<unique customer id in your system>\",\n        \"name\": {\n            \"givenName\": \"John\",\n            \"middleNames\": [\n                \"Jay\"\n            ],\n            \"familyName\": \"Doe\"\n        },\n        \"dateOfBirth\": \"1977-07-01\",\n        \"address\": {\n            \"firstLine\": \"53 Shoreditch \",\n            \"city\": \"Tallinn\",\n            \"countryCode\": \"EE\",\n            \"postCode\": \"12112\"\n        }\n    }\n}",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json:1069]: `originalTransactionId` in the docs we say this is the ID in their system I think?

We have `reference` twice.

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 12]
```diff
@@ -0,0 +1,1728 @@
...
+									"options": {
+										"raw": {
+											"language": "json"
+										}
+									}
+								},
+								"url": {
+									"raw": "{{host}}/v1/profiles/{{personal-profile-id}}/third-party-transfers",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
									"raw": "{{host}}/v1/profiles/{{business-profile-id}}/third-party-transfers",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 13]
```diff
@@ -0,0 +1,1728 @@
...
+									"key": "token",
+									"value": "{{token}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "https://api.transferwise.com/v1/profiles/{{personal-profile-id}}/transfers/{{new-transfer-id}}/deposit-details/bank-transfer",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
							"raw": "{{host}}/v1/profiles/{{business-profile-id}}/transfers/{{new-transfer-id}}/deposit-details/bank-transfer",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 14]
```diff
@@ -0,0 +1,1728 @@
...
+							"protocol": "https",
+							"host": [
+								"api",
+								"transferwise",
+								"com"
+							],
+							"path": [
+								"v1",
+								"profiles",
+								"{{personal-profile-id}}",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
								"{{business-profile-id}}",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 15]
```diff
@@ -0,0 +1,1728 @@
...
+						},
+						"url": {
+							"raw": "{{host}}/v3/profiles/{{personal-profile-id}}/transfers/{{new-transfer-id}}/payments",
+							"host": [
+								"{{host}}"
+							],
+							"path": [
+								"v3",
+								"profiles",
+								"{{personal-profile-id}}",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
								"{{business-profile-id}}",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 16]
```diff
@@ -0,0 +1,1728 @@
...
+							"mode": "raw",
+							"raw": "{\n    \"type\": \"BALANCE\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{host}}/v3/profiles/{{personal-profile-id}}/transfers/{{new-transfer-id}}/payments",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
							"raw": "{{host}}/v3/profiles/{{business-profile-id}}/transfers/{{new-transfer-id}}/payments",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 17]
```diff
@@ -0,0 +1,1728 @@
...
+								},
+								"url": {
+									"raw": "{{host}}/v1/profiles/{{personal-profile-id}}/third-party-transfers",
+									"host": [
+										"{{host}}"
+									],
+									"path": [
+										"v1",
+										"profiles",
+										"{{personal-profile-id}}",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
										"{{business-profile-id}}",
```

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 18]
```diff
@@ -0,0 +1,1728 @@
...
+										"third-party-transfers"
+									]
+								}
+							},
+							"response": []
+						}
+					],
+					"event": [
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: remove?

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 19]
```diff
@@ -0,0 +1,1728 @@
...
+							},
+							"request": {
+								"method": "GET",
+								"header": [
+									{
+										"key": "Authorization",
+										"value": "Bearer {{token}}"
+									}
+								],
+								"body": {
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: remove? Maybe doesn't matter

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 20]
```diff
@@ -0,0 +1,1728 @@
...
+									"mode": "formdata",
+									"formdata": []
+								},
+								"url": {
+									"raw": "{{host}}/v1/quotes/{{new-quote-id}}/pay-in-methods",
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json:1115]: Should this definitely be v1 even for a v2 quote? quite confusing 😞

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 13 above)
Comment (JamesFerretti) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Also should this definitely be v1 too? We tend to use v3 for any /profiles/ endpoint now.

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 21]
```diff
@@ -0,0 +1,1728 @@
...
+							"raw": "{{host}}/oauth/token",
+							"host": [
+								"{{host}}"
+							],
+							"path": [
+								"oauth",
+								"token"
+							]
+						},
+						"description": "Get a client credentials token to use to create a user"
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: This is the wrong description

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json [hunk 22]
```diff
@@ -0,0 +1,1728 @@
...
+							],
+							"path": [
+								"v3",
+								"applications",
+								"{{client-id}}",
+								"subscriptions",
+								""
+							]
+						},
+						"description": "Subscribe application for transfer status change events"
```
Comment (JamesFerretti) [inline payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]:
```suggestion
						"description": "Subscribe to transfer status change events for transfers created using this application"
```

**Review by JamesFerretti — CHANGES_REQUESTED** [review]

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 11 above)
Comment (sanj-kv) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json:1069]: There is a reference for details - where the user can mention what is this transfer for. Then, the other reference is the originator's ID. 
originalTransactionId - that is the guid the partner creates. That should be the ID in their system?

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 20 above)
Comment (sanj-kv) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json:1115]: @JamesFerretti , for this, there is no v2.

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 7 above)
Comment (sanj-kv) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: @JamesFerretti Refund recipient is not part of the payouts guide. Should we add that there?

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 13 above)
Comment (sanj-kv) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: @JamesFerretti This is surely only v1. There is no v2, v3 for this endpoint sadly.

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 20 above)
Comment (JamesFerretti) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json:1115]: Cool

File: payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json (hunk 7 above)
Comment (JamesFerretti) [reply payouts-and-third-party-payments/Payouts and third party payouts API.postman_collection.json]: Yes, I think we should add it, surely there are cases we need it? E.g Brex

**Review by JamesFerretti — APPROVED** [review]

**Pull request merged**